packages/platforms/agema/x86-64/x86-64-agema-agc7648a/platform-config/r0/src/lib/x86-64-agc7648a-r0.yml

```diff
@@ -18,7 +18,7 @@ x86-64-agc7648-r0:
       --parity=0
       --stop=1
 
-    kernel: 
+    kernel:
```

Comment:
@spitchai  The 3.18 kernel is going to be deprecated. Only the 3.16 and future 4.X kernels will be supported. 
Can you use the 3.16 kernel instead?